sw/device/silicon_creator/lib/chip_info.h

```diff
@@ -0,0 +1,26 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_CHIP_INFO_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_CHIP_INFO_H_
+
+#include <assert.h>
+#include <stdint.h>
+
+typedef struct chip_info {
+  uint32_t hw_rev;
+  uint64_t scm_revision;
+  char build_date_str[20];
```

Comment:
If you were to order this as (`uint64_t`, `uint32_t`, `char[20]`), the size of this structure would be 32 bytes instead of 40 and would have no undeclared padding.  As it is currently, there is 4 bytes of padding between `hw_rev` and `scm_revision` and 4 bytes of padding after `build_date_str` so the struct will align properly in an array.

Reply:
Thanks! That makes sense. Just updated the PR.